js/index.js

```diff
@@ -1,6 +1,144 @@
+Vivamus vitae tristique dui. Morbi quam felis, elementum et tortor non, rutrum sodales dolor. Pellentesque a egestas lacus. Cras laoreet neque vel rhoncus sodales. Sed vel velit enim. Aenean in eros id odio mollis blandit ut ac est. Etiam maximus lectus eu ullamcorper pharetra.
+Curabitur quis dui quis tortor mollis dictum in sed turpis. Duis id velit at sem egestas vehicula sed ac justo. Aenean egestas sodales nunc ac interdum. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec congue eget lectus eu dignissim. Curabitur ac metus at odio ultrices facilisis sit amet ut tellus. Quisque eu nunc hendrerit, vestibulum lacus non, lobortis ipsum.`
+let wordCount = 0
+for (let i = 0; i < paragraphs.length; i++){
+  let nextLetter = i + 1
+    if(paragraphs[i] === " "){
+      wordCount++
+    }
+}
+console.log(wordCount)
+// this produced a count short of the true number - as it missed the first words of the paragraph 
+// we then found the following way to create a word count
+
+let newWordCount = paragraphs.match(/(\w+)/g).length
+
+console.log(newWordCount)
+
+// Make your program count the number of times the Latin word et appears.
+let count = 0
+
+for (let i = 0; i < paragraphs.length; i++){
+  if (paragraphs[i] === ' '){
+    if (paragraphs[i + 1] === 'e'){
+      if (paragraphs[i + 2] === 't'){
+        count++
+      }
+    }
+  }
+}
++ 
+console.log(count)
+
+//Bonus 2
+// Create a new variable phraseToCheck and have it contain some string value. Write a code that will check if the value we assigned to this variable is a Palindrome. Here are some examples of palindromes:
+/* 
+"A man, a plan, a canal, Panama!"
+"Amor, Roma"
+"race car"
+"stack cats"
+"step on no pets"
+"taco cat"
+"put it up"
+"Was it a car or a cat I saw?" and 
+"No 'x' in Nixon".
+*/
+
+let phraseToCheck = "No 'x' in Nixon"
+let backwardsWord = ""
+let spacelessWord = ""
+let isPalindrome
+
+for (let i = phraseToCheck.length - 1; i >= 0; i--){
+  if (phraseToCheck[i] === " "){
+  } else if (phraseToCheck[i] === ","){
+    continue
+  } else if (phraseToCheck[i] === "!"){
+    continue
+  } else if (phraseToCheck[i] === "?"){
+    continue
+  } else if(phraseToCheck[i] === "'"){
+    continue
+  } else {
+    backwardsWord += phraseToCheck[i]
+  }
+}
+
+for (let i = 0; i < phraseToCheck.length; i++){
+  if (phraseToCheck[i] === " "){
+  } else if (phraseToCheck[i] === ","){
+    continue
+  } else if (phraseToCheck[i] === "!"){
+    continue
+  } else if (phraseToCheck[i] === "?"){
+    continue
+  } else if(phraseToCheck[i] === "'"){
+    continue
+  } else {
+    spacelessWord += phraseToCheck[i]
+  }
+}
+  backwardsWord.toLowerCase()
+  phraseToCheck.toLowerCase()
+
+if (spacelessWord.toLowerCase === backwardsWord.toLowerCase){
+  isPalindrome = true
+} else {
+  isPalindrome = false
+}
```

Comment:
Excellent work both!

The only note here. Currently in bonus2 solution you always get true. 

On line 136, inside if statement you forgot to add bracket.
- `toLowerCase() `is a method (function), so you have to 'call it' to use it. 
-  Otherwise you are just comparing two functions literally , but not the result of each function, not the string in lower case